IC-benchmarking-env.yml

```diff
@@ -0,0 +1,15 @@
+#modeled off of  "NF1_SchwannCell_data_analysis/environment.yml" & "nf1_cellpaining_data/nf1_cellpainting_env.yml"
+name: IC-benchmarking-env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.9
+  - conda-forge::pip
+  - conda-forge::numpy
+  - conda-forge::nb_conda_kernels
+  - conda-forge::pycytominer
+  - pip:
+    - cellprofiler>=4.2.4
```

Comment:
Just as an FYI, I forgot to update the NF1 env you might have seen, but 4.2.4 is the env version we normally use. The > I noticed led to some fun issues that you must have resolved. 4.2.6 will work just fine, just wanted to provide some context :smile:

Reply:
Ah I see. Should I do cellprofiler=4.2.6 instead so that it is clear what version I am using? Also I believe I made all the other changes. Thanks!

Reply:
Probably include as a comment above it would work!